lua/nvim-tree/legacy.lua

```diff
@@ -285,26 +285,14 @@ local function refactored(opts)
     end
   end
 
-  -- update_to_buf_dir -> hijack_directories
-  if opts.update_to_buf_dir ~= nil then
-    utils.table_create_missing(opts, "hijack_directories")
-    if opts.hijack_directories.enable == nil then
-      opts.hijack_directories.enable = opts.update_to_buf_dir.enable
-    end
-    if opts.hijack_directories.auto_open == nil then
-      opts.hijack_directories.auto_open = opts.update_to_buf_dir.auto_open
-    end
-    opts.update_to_buf_dir = nil
-  end
+  utils.move_missing_val(opts, "update_to_buf_dir", "enable", opts, "hijack_directories", "enable")
+  utils.move_missing_val(opts, "update_to_buf_dir", "auto_open", opts, "hijack_directories", "auto_open")
```

Comment:
i think this change can be removed alltogether, its been quite a while since we've renamed update_to_buf_dir to hijack_directories.

Reply:
I don't think we can do that. The users have not been told that the rename has happened.

If we do remove, the users will get a warning and that option will be removed. They would need to search help to see what the change is.

The `g:` we can remove soon, as we have explicitly warned the users.

Reply:
although this was renamed in february so it's been quite a while. It's also present in the migration guide and removing it would just print a warning to the user.

Reply:
removed